pkg/placementrequests/v1alpha1/placementrequests.go

```diff
@@ -64,3 +64,27 @@ func SetPodBindingResult(
 		},
 	)
 }
+
+// AssessResult role is to assess, based on the placement request status, if
+// it was successful or not. This function returns the result and a human
+// readable message.
+func AssessResult(pr *v1alpha1.PlacementRequest) (v1alpha1.PlacementRequestResult, string) {
```

Comment:
Could be helpful to write a unit test for the new method. So things don't break that easily.

Reply:
Done. Please take a look.